src/app/Library/CrudPanel/Uploads/Uploaders/SingleBase64Image.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Backpack\CRUD\app\Library\CrudPanel\Uploads\Uploaders;
+
+use Backpack\CRUD\app\Library\CrudPanel\CrudPanelFacade as CRUD;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Str;
+
+class SingleBase64Image extends Uploader
+{
+    public function save(Model $entry, $value = null)
+    {
+        return $this->isRepeatable && ! $this->isRelationship ? $this->saveRepeatableSingleBase64($entry, $value) : $this->saveSingleBase64($entry, $value);
```

Comment:
I saw this conditional a lot, perhaps we can create one an re-use it? `$this->isRepeatableButNotRelationship()`

Reply:
Alternatively... Can an uploader NOT support relationships? 

In the end I see all uploaders have the `save()` method looking the same: if repeatable do smth, else do smth else. So maybe... we should have the default `save()` method do that, and call two separate methods for each?

```
public function save($entry, $value) {
        return $this->isRepeatable && ! $this->isRelationship ? $this->saveRepeatable($entry, $value) : $this->saveSingle($entry, $value);
}
```

and then the developers' responsibility would be to define these two methods... `saveSingle()` and `saveRepeatable()` right?

And even if there were an Uploader that didn't support repeatable... the `save()` function could account for that right? If `saveRepeatable()` hasn't been defined... just use the regular one.

Reply:
Also... does this actually SAVE or does it STORE something? 👀 Cause from what I see in the code, it uploads the file to disk, which I'd call `store()` not `save()`. Or better yet... you know what I'd call it? 😅 `upload()` 😅